infra/core/monitor/applicationinsights.bicep

```diff
@@ -1,9 +1,8 @@
 metadata description = 'Creates an Application Insights instance based on an existing Log Analytics workspace.'
 param name string
-param dashboardName string
+param dashboardName string = ''
```

Comment:
Can we get this into core?

Reply:
Yep PR incoming

Reply:
https://github.com/Azure/azure-dev/pull/3009